engine/rowset/src/main/java/io/deephaven/engine/rowset/impl/singlerange/SingleRange.java

```diff
@@ -22,6 +22,10 @@
 import java.util.function.LongConsumer;
 
 public abstract class SingleRange implements OrderedLongSet {
+
+    private static final boolean debug =
```

Comment:
I think the finer-grained debugging has value. We already had multiple debug flags, suggesting a need for fine-grained debugging in past cases. I would somewhat rather keep these separate, since you can easily combine multiple flags but you can't easily split them again.